src/unit_tests/unit_tests.cmake

```diff
@@ -39,7 +39,7 @@ set(SOURCE_FILES main.cpp
                  misc/array_range_test.cpp
                  misc/string_view_test.cpp
                  misc/enum_flags_test.cpp
-                 misc/library_test.cpp
+                 # misc/library_test.cpp After exporting the registration manager, the library test does not make sense any more.
```

Comment:
I don't know why you think it doesn't make any sense anymore. This plugin library loading functionality that is heavily used by OMX and RTX